fontface.go

```diff
@@ -227,22 +227,11 @@ func (ff FontFace) Metrics() FontMetrics {
 
 // Kerning returns the kerning between two runes in mm (ie. the adjustment on the advance).
 func (ff FontFace) Kerning(rPrev, rNext rune) float64 {
-	buffer := &sfnt.Buffer{}
-	prevIndex, err := ff.font.sfnt.GlyphIndex(buffer, rPrev)
-	if err != nil {
-		return 0.0
-	}
-
-	nextIndex, err := ff.font.sfnt.GlyphIndex(buffer, rNext)
+	k, err := ff.Font.Kerning(rPrev, rNext, ff.Size*ff.Scale)
```

Comment:
If we're sure that `k == 0.0` when `err != nil` than I'd agree we can just return `k` but a comment would be nice.

Reply:
I added a comment at the method level